src/main/target/common_defaults_post.h

```diff
@@ -316,6 +316,13 @@
 #endif
 #endif
 
+// F4 and F7 single gyro boards
+#if defined(USE_MULTI_GYRO) && !defined(GYRO_2_SPI_INSTANCE)
+#define GYRO_2_SPI_INSTANCE     GYRO_1_SPI_INSTANCE
```

Comment:
Why was this moved up? The ordering of these is intentional, and designed to not break in the case of `GYRO_1_SPI_INSTANCE` not being defined.

Reply:
I think that is the result of a rebase since this code was written back in January.  Will investigate and fix.

Reply:
ahh yes, it was moved up because the old code defined an alignment for GYRO_2 and now the alignment for gyro is done separately, however I missed the case you mention so I will re-arrange it some.

Reply:
however, after re-reading it, it doesn't matter since the macro for `GYRO_2_SPI_INSTANCE` will be evaluated to `NULL` by the check for `GYRO_1_SPI_INSTANCE` or whatever the value of `GYRO_1_SPI_INSTANCE` is already set to.